### PR TITLE
bug fix: spell checking and typing errors

### PR DESCRIPTION
**Pull request by Danege**

### CODE REVIEW

File: web-security/introduction-securite-web-pour-futurs-developpeurs-short.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ Mais aussi parce que réfléchir aux problématiques de la sécurité pour vos p
 
 --------------------------------------------------------------------------------
 
-# Amusant?
+# Amusant ?
 
-.notes: ...Oui
+.notes : ...Oui
```
Comment (regilero) [inline web-security/introduction-securite-web-pour-futurs-developpeurs-short.md]: Not sure this is supported, here it's not a french spacing, that's a 'language feature' and it should stay an english spacing (no space)

File: web-security/introduction-securite-web-pour-futurs-developpeurs-short.md [hunk 2]
```diff
@@ -763,7 +763,7 @@ Ce principe est le deuxième grand principe (après les validations et filtrages
 > Soyez paranoïaques, on parle de systèmes automatisés, ces systèmes ont très peu de bon sens, ils peuvent laisser entrer un éléphant dans votre salon si celui-ci conduit votre voiture, porte votre cravate et possède les clefs de la maison.
 ![BANKSY](./banksy.jpg)
 
-(photo elephant: banksy)
+(photo elephant : banksy)
```
Comment (regilero) [inline web-security/introduction-securite-web-pour-futurs-developpeurs-short.md]: same remark: this one should stay with english no-space

**Comment by regilero** [comment]
thanks a lot for all this stuff, could you please just remove the two commented spaces and I'll merge this.

Really, that's nice, thanks.

**Comment by Danege** [comment]
fixed

**Comment by regilero** [comment]
That's merged, thanks again.

**Pull request merged**